package.json

```
@@ -7,6 +7,8 @@
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev --parallel",
+    "docs": "npm-run-all -l build --parallel dev docs:run",
+    "docs:run": "cd docs && node ../packages/mystmd/dist/myst.cjs start .",
```

Does it need the '.'?

```suggestion
    "docs:run": "cd docs && node ../packages/mystmd/dist/myst.cjs start",
```